common/src/stack/command/stack/commands/add/pallet/plugin_remote_path.py

```diff
@@ -56,7 +56,7 @@ def run(self, args):
 			wget_cmd = [
 				'wget',
 				*creds,
-				'--quiet',
+				'--no-verbose',
```

Comment:
Consider whether someone debugging this in the future would appreciate a suppressed level of output.